### ISSUE TITLE
Implement Dice + Cross Entropy loss from 'No New Net' paper

### ISSUE THREAD

**Issue by Zach-ER** [NifTK/NiftyNet#241]
The Medical Imaging Decathlon was won by [Isensee et al](https://arxiv.org/pdf/1809.10486.pdf). They used a loss function that is the sum of the Dice loss and Cross Entropy. This should be implemented as an option for the segmentation application.
